src/controllers/user.js

```diff
@@ -86,15 +81,17 @@ v1.put =
     user = await users.findOne({ userId: userId })
     createP = !user
     if (createP) {
-      // NB: payload.version should be mandatory
-      if ((payload.version) && (payload.version !== 1)) {
+      if ((!payload.version) && (payload.version !== 1)) {
```

Comment:
shouldn't this be `if (!payload.version || payload.version !==1)`? can be simplified to just `if (payload.version !==1)`.